odbc/Cargo.toml

```diff
@@ -32,4 +32,4 @@ thiserror = "1"
 
 [lib]
 name = "mongoodbc"
-crate-type = ["cdylib"]
+crate-type = ["cdylib","lib"]
```

Comment:
I added `lib` here so that `mongoodbc` could be imported from the odbc/tests/ directory.